opentelemetry-sdk/src/metrics/internal/aggregate.rs

```diff
@@ -210,3 +210,239 @@ impl<T: Number<T>> AggregateBuilder<T> {
         )
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/issues/1432 Would it makes sense to add more e2e tests as well from this issue description? Its still very basic with lots of copy-pastes.. still good to verify these issues.

Reply:
Could always have more test coverage, but this removes the issue for aggregates that the issue is referencing by not re-using the data point cells entirely so it's lower risk.